src/components/ServiceArea.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const ServiceArea = () => {
+  return (
+    <div className='bg-gray-100'>
+      <div className='px-6 pt-10'>
+        <h2 className='text-center mb-3 text-5xl md:text-6xl font-header'>Areas We Serve</h2>
```

Comment:
Note (nb): I'm using text-4xl md:text-6xl at the moment. Not sure if we have to use the same size as it is a page title. Could we mention this to the designer?